src/nunit-gui/Presenters/TestPropertiesPresenter.cs

```diff
@@ -118,6 +114,19 @@ private void DisplayTestProperties()
                 _view.Header = testItem.Name;
             }
         }
+
+        public static string GetTestType(TestNode testNode)
+        {
+            if (testNode.RunState == RunState.NotRunnable)
```

Comment:
This also needs to be conditioned on testNode.Type == "Assembly" A non-runnable Fixture, for example, is still a Fixture.